src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipIncorporationService.java

```diff
@@ -89,16 +92,18 @@ private Resource createIncorporationTransactionResource(String incorporationUri,
         var incorporationResource = new Resource();
 
         Map<String, String> linksMap = new HashMap<>();
-        linksMap.put("resource", incorporationUri);
-        linksMap.put("costs", incorporationUri + "/costs");
+        linksMap.put(LINK_RESOURCE, incorporationUri);
+
+        if (IncorporationKind.REGISTRATION.getDescription().equals(kind)) {
```

Comment:
would it be better to move the description check into TransactionUtils and have a isForRegistration(transaction) method

Reply:
Yep. Might be more readable.

Reply:
I think it's still worth doing like you suggest, as check is repeated in the other services. Just need to pass `transaction` into the method.